golden_bridge.rb

```diff
@@ -0,0 +1,61 @@
+class Array
+  def convert_to_integer
+    return [] if empty?
+
+    map(&:to_i)
+  end
+
+  def quicksort
```

Comment:
Why didn't you use the `sort` method that is predefined in the array class. Something like: `[3, 2, 1].sort`.